packages/sql/src/RequestManager.ts

```diff
@@ -13,7 +13,7 @@ interface FetchArgs {
 
 const UNKNOWN = -1;
 const NO_RETRY = -1;
-const MAX_RETRIES = 5;
+const MAX_RETRIES = 0;
```

Comment:
maybe call this DEFAULT_MAX_RETRIES?

Reply:
Changed all to `DEFAULT_MAX_API_REQUESTS_RETRIES` and all files get it from SQL module